lib/castle/application.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Castle.Application do
+  use Application
+
+  # See http://elixir-lang.org/docs/stable/elixir/Application.html
+  # for more information on OTP Applications
+  def start(_type, _args) do
+    import Supervisor.Spec
+
+    # Define workers and child supervisors to be supervised
+    children = [
+      supervisor(Castle.Repo, []),
+      supervisor(CastleWeb.Endpoint, []),
+      worker(Castle.Scheduler, []),
```

Comment:
In more recent versions of Phoenix, this app config moves to the `application.ex` file.  Added a supervisor for the Ecto repo, and a worker for the Quantum scheduler.  Otherwise stayed the same.